bot/cogs/utility/nickrequest.py

```diff
@@ -6,8 +6,8 @@
 from config.config import nick_request_channel_id as request_channel_id
 from config.config import nick_accept_channel_id as accept_channel_id
 
-reaction_timeout = 300
 command_timeout = 600
+allowed_list = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;?@[\\]^_`{|}~ '
```

Comment:
```python
import string
allowed_list = string.digits + string.letters + string.punctuation + " "
```

Reply:
Thanks, will implement in a fix soon:tm: